<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResourceBrowser</class>
 <widget class="QDialog" name="ResourceBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resource Browser</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="intro_label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:xx-large; font-weight:600;&quot;&gt;Yonder Resource Browser&lt;/span&gt;&lt;/p&gt;&lt;p&gt;If you know of any other good resource, please tell us about it in our forums at &lt;a href=&quot;http://yonderfx.com/forums/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;yonderfx.com!&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Though we try to keep these links up to date, there is no guarantee that their content is not outdated and/or inappropriate&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEditFilter" name="lineedit_search">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Filter</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkbox_loop">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Loop</string>
         </property>
         <property name="tristate">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkbox_music">
         <property name="text">
          <string>Music</string>
         </property>
         <property name="tristate">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkbox_sfx">
         <property name="text">
          <string>SFX</string>
         </property>
         <property name="tristate">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combobox_license">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <property name="minimumContentsLength">
          <number>15</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QListView" name="resource_view">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="resource_data_view">
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
         <property name="openLinks">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLineEditFilter</class>
   <extends>QLineEdit</extends>
   <header>view/qlineeditfilter.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineedit_search</tabstop>
  <tabstop>resource_view</tabstop>
  <tabstop>checkbox_loop</tabstop>
  <tabstop>checkbox_music</tabstop>
  <tabstop>checkbox_sfx</tabstop>
  <tabstop>combobox_license</tabstop>
  <tabstop>resource_data_view</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
